packages/web-components/src/components/ic-select/ic-select.tsx

```diff
@@ -220,7 +220,7 @@ export class Select {
         this.charactersUntilSuggestions === 0) ||
       (this.searchableSelectInputValue &&
         this.searchableSelectInputValue.length >=
-          this.charactersUntilSuggestions)
+        this.charactersUntilSuggestions)
```

Comment:
Not sure if we're getting some inconsistencies with npm run prettier:fix but for me running it adds this indentation back in, same with other things like line 705 🤔

Reply:
I ran it again and it seems to have added it back in. A bit weird that it changes each time I run it, but hopefully it's not a consistent issue